<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgressBarWidget</class>
 <widget class="QWidget" name="ProgressBarWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>70</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="textDirection">
    <enum>QProgressBar::TopToBottom</enum>
   </property>
   <property name="format">
    <string>%p%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Msg">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Message Label</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_CurrentIndex">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Now at :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_CurrentIndex">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_ChangeTo">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Change to</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
